mrpt_ro_slam/src/mrpt_ro_slam/mrpt_ro_slam.cpp

```diff
@@ -67,10 +67,12 @@ using namespace mrpt::obs;
 
 RBPFSlam::~RBPFSlam()
 {
+	delete mapBuilderRBPF;
 }
 
 RBPFSlam::RBPFSlam(Parameters *param)
     : RBPFSlamCore(), param_(param) {
+	
```

Comment:
You could also add here to the init list something like: 

        , mapBuilderRBPF(NULL)

to make sure it gets initialized to NULL, so it can be detected if the user calls to whatever function before the object is really build in `::init()`, etc.